app/views/qsets/_question_list_filter.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class="row question_list">
+
+  <div class="panel-group no-questions hidden panel panel-default" role="dialog">
```

Comment:
No need for `panel-group` class here -- or anywhere in this file, in fact. Might as well clean it up. (for more info on panel group, and why it doesn't apply here, see http://www.w3schools.com/bootstrap/bootstrap_panels.asp) -- so please remove all panel-group classes (and the comment on line 79) and verify that it doesn't change the styles (it doesn't when I test it, but you should test it too). Thanks!